gulpfile.js

```diff
@@ -6,12 +6,28 @@ const sass = require('gulp-sass');
 const svgSprite = require('gulp-svg-sprite');
 const swBuild = require('sw-build');
 const fs = require('fs');
+const path = require('path');
```

Comment:
We now have a lower case "path" and a "Path" variable in the same file. Can you rename them so they make more sense in context?